app/controllers/decidim/participations/admin/participations_controller.rb

```diff
@@ -6,8 +6,16 @@ module Admin
       # This controller allows admins to manage participations in a participatory process.
       class ParticipationsController < Admin::ApplicationController
         helper Participations::ApplicationHelper
+        helper Decidim::Messaging::ConversationHelper
+
         helper_method :participations, :query
 
+        def index
+          @param_unmoderate = true if params[:status].nil? || params[:status] == "unmoderate"
```

Comment:
would it be better written with a `switch case` for readability?

Reply:
I am just wondering here, not saying it's necessary better.
But you could also have only stored the status value (ie. `@status`) and switch case it in `query`.
I am not sure what benefits it would provide